test/completion.js

```diff
@@ -121,13 +121,20 @@ describe('Completion', function () {
       // SKipping on CI right now, otherwise might introduce an `npm install
       // generator-dummy` as a pretest script
 
-      it.skip('Returns the list of user installed generators as completion candidates', function (done) {
+      before(function(done) {
```

Comment:
@SBoudrias Hope it's ok to add these global installs / uninstall. This way I can test out a real completion output for a given generator.